docker/Dockerfile-workers

```diff
@@ -24,7 +24,7 @@ RUN dnf -y install \
     && dnf update -y \
     && dnf clean all
 
-ADD https://github.com/operator-framework/operator-registry/releases/download/v1.19.5/linux-amd64-opm /usr/bin/opm
+ADD https://github.com/operator-framework/operator-registry/releases/download/v1.20.0/linux-amd64-opm /usr/bin/opm
```

Comment:
Let's put there latest one since it is not merged yet: 
https://github.com/operator-framework/operator-registry/releases/tag/v1.21.0

Reply:
Agreed.

I'll change it and push again